Views/Card/Index.cshtml

```diff
@@ -9,12 +9,12 @@
 <h1>Your card(s):</h1>
 
 <div>
-    @{
-        foreach (Card card in Model)
-        {
-            <img src="@card.Image.GetPNGString()" width="250" height="363" />
```

Comment:
Complete removal of width and height settings isn't desired. Height and width will be added as parameters for ease of modification.